Exercise_B/index.html

```diff
@@ -4,31 +4,58 @@
+    <link rel="stylesheet" href="/Exercise_B/assets/style/stylesheet.css">
     <!--If you want to use bootstrap properly this is vital (helps browser to measure the page and readjust the information accordingly-->
-
     <!-- Link and CDN required for bootstrap -->
-
     <!-- Google Font - Lato -->
     <link
       href="https://fonts.googleapis.com/css?family=Lato"
       rel="stylesheet"
     />
     <!-- Custom Stylesheet -->
-    <link rel="stylesheet" href="./assets/style/stylesheet.css" />
+    <!-- <link rel="stylesheet" href="./assets/style/stylesheet.css" /> -->
   </head>
 
   <body>
-    <!--Header (flower shop logo block element)-->
-    <!--Banner Jumbotron of the image-->
+        <!--Header (flower shop logo block element)-->
+    <h1>
+      <img src="/Exercise_B/assets/images/logo.png" alt="logo">
+    </h1>
+        <!--Banner Jumbotron of the image-->
+    <div class="brightness(0.4)">
+      <img class="img-fluid w-100" src="/Exercise_B/assets/images/flowershop.jpg" alt="flowershop">
+        <h1 class="centered">Beautiful Flowers</h1>
+        <button class="pressme" type="button" onclick="alert('Flowers are Cool!')">Learn More!</button>
+    </div>
 
     <!--Four containers with img / text-->
     <!-- 4 containers -->
+  <section class="container-fluid p-5 bg-success">
+    <div class="d-flex flex-row justify-content-center flex-column d-md-flex flex-md-row">
+      <img src="/Exercise_B/assets/images/flower-icon.png" alt="flowerIcon">
+        <p>Flowers for every kind of situation</p>  
+          <!-- Payments Div -->
+      <div class="d-flex flex-row justify-content-center flex-column d-md-flex flex-md-row">
+        <img src="/Exercise_B/assets/images/payment-icon.png" alt="paymentIcon">
+          <p>Flowers for every kind of situation</p>  
+      </div>
+          <!-- Consultation Div -->
+      <div class="d-flex flex-row justify-content-center flex-column d-md-flex flex-md-row">
+        <img src="/Exercise_B/assets/images/phone-icon.png" alt="phoneIcon">
+          <p>Flowers for every kind of situation</p>  
+      </div>
+          <!-- Delivery Div -->
+      <div class="d-flex flex-row justify-content-center flex-column d-md-flex flex-md-row">
+        <img src="/Exercise_B/assets/images/truck-icon.png" alt="truckIcon">
+          <p>Flowers for every kind of situation</p>  
+      </div>
+    </div>
 
-    <!-- Payments Div -->
-    <!-- Consultation Div -->
-
-    <!-- Delivery Div -->
+  </section>
 
     <!--Scripts required for bootstrap CDN-->
+    <script src="https://cdn.jsdelivr.net/npm/@popperjs/core@2.9.3/dist/umd/popper.min.js" integrity="sha384-eMNCOe7tC1doHpGoWe/6oMVemdAVTMs2xqW4mwXrXsW0L84Iytr2wi5v2QjrP/xp" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.0/dist/js/bootstrap.min.js" integrity="sha384-cn7l7gDp0eyniUwwAZgrzD06kc/tftFf19TOAs2zVinnD/C7E91j9yyk5//jjpt/" crossorigin="anonymous"></script>
   </body>
 </html>

```

Comment:
For referencing the image, try to use the ../ so that it can be linked to the image and then loaded on the page. In terms of design it's done well, the only issue is that the images were not being resized according to the device. CSS could be used for this part to resize the image and add flex to the container of items.